sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_directory_client.py

```diff
@@ -102,8 +102,8 @@ def from_connection_string(
             an instance of a AzureSasCredential from azure.core.credentials, and account shared access
             key, or an instance of a TokenCredentials class from azure.identity.
             Credentials provided here will take precedence over those in the connection string.
-        :return a DataLakeDirectoryClient
-        :rtype ~azure.storage.filedatalake.DataLakeDirectoryClient
+        :returns: a DataLakeDirectoryClient
```

Comment:
So, I know you mentioned it in the PR description but a bit more on `:return:` versus `:returns:`. Looking at the [generated docs](https://docs.microsoft.com/en-us/python/api/azure-storage-file-datalake/azure.storage.filedatalake.datalakedirectoryclient?view=azure-python#azure-storage-filedatalake-datalakedirectoryclient-create-directory), it seems both will actually work.

That being said, since the generated code and the majority of the places you touched here seemed to go with `:return:`, I think we should go with that (sorry! since you chose the other one).

Good catch on the colons here though, the docs for this method were quite messed up without them :).